testdata/jennies/rawtypes/struct_with_complex_fields/PythonRawTypes/models/struct_complex_fields.py

```diff
@@ -14,8 +14,9 @@ class SomeStruct:
     field_array_of_strings: list[str]
     field_map_of_string_to_string: dict[str, str]
     field_anonymous_struct: 'StructComplexFieldsSomeStructFieldAnonymousStruct'
+    field_ref_to_constant: 'ConnectionPath'
 
-    def __init__(self, field_ref: typing.Optional['SomeOtherStruct'] = None, field_disjunction_of_scalars: typing.Union[str, bool] = "", field_mixed_disjunction: typing.Union[str, 'SomeOtherStruct'] = "", field_disjunction_with_null: typing.Optional[str] = None, operator: typing.Optional[typing.Literal[">", "<"]] = None, field_array_of_strings: typing.Optional[list[str]] = None, field_map_of_string_to_string: typing.Optional[dict[str, str]] = None, field_anonymous_struct: typing.Optional['StructComplexFieldsSomeStructFieldAnonymousStruct'] = None):
+    def __init__(self, field_ref: typing.Optional['SomeOtherStruct'] = None, field_disjunction_of_scalars: typing.Union[str, bool] = "", field_mixed_disjunction: typing.Union[str, 'SomeOtherStruct'] = "", field_disjunction_with_null: typing.Optional[str] = None, operator: typing.Optional[typing.Literal[">", "<"]] = None, field_array_of_strings: typing.Optional[list[str]] = None, field_map_of_string_to_string: typing.Optional[dict[str, str]] = None, field_anonymous_struct: typing.Optional['StructComplexFieldsSomeStructFieldAnonymousStruct'] = None, field_ref_to_constant: typing.Optional['ConnectionPath'] = None):
```

Comment:
Could this line be broken into multiple lines so it becomes a bit easier to read?

Reply:
Not without implementing this feature in the Python jenny :/

This file is used in tests, as a reference for what that jenny generates